src/main/resolve/server.ts

```diff
@@ -41,21 +41,31 @@ export function findAvailablePort(startPort: number): Promise<number> {
   })
 }
 
+let pacServer: http.Server
+
 export async function startPacServer(): Promise<void> {
```

Comment:
The server should be stop before each start.

Reply:
fixed